packages/cli/src/index.ts

```diff
@@ -16,6 +16,17 @@ const packageJson = require('../package.json');
 
 import program = require('commander');
 
+const MINUMUM_NODE_VERSION = 12;
+
+const [nodeMajorVersion] = process.versions.node.split('.');
+if (parseInt(nodeMajorVersion, 10) < MINUMUM_NODE_VERSION) {
+  console.error(`\x1b[4m\x1b[31mYou have an outdated node version!`);
```

Comment:
maybe add those as constants (or use a color package like https://www.npmjs.com/package/colors)

```
const underline = s =>  `\x1b[4m\x1b[31m${s}`

// ...
console.error(underline('You have an outdated node version!')
```

bit easier to read and also reusable

Reply:
Hmm, thing is we normally using react ink for all that. But because this is before this is set up I did it like this.

Reply:
I just made the formatting constants, then it's easier to read without adding more code complexity for something we only need in one place